packages/interim/src/renovation/component_wrapper/common/mutations_recording.ts

```diff
@@ -1,27 +1,26 @@
 function isChildNode(node: Node): node is ChildNode {
-    return typeof (node as Partial<ChildNode>).remove === 'function';
+  return typeof (node as Partial<ChildNode>).remove === 'function';
+}
+
+function revertMutation({ type, addedNodes }: MutationRecord): void {
+  switch (type) {
+    case 'childList':
+      addedNodes.forEach((n) => isChildNode(n) && n.remove());
+      break;
+    default:
+      break;
   }
```

Comment:
Minor suggestion.

Let's don't use the ```switch``` with one possible option:
```suggestion
function revertMutation({ type, addedNodes }: MutationRecord): void {
  if (type === 'childList') {
    addedNodes.forEach((n) => isChildNode(n) && n.remove());
  }
}
```

Reply:
I like using `switch` statement in such cases because it highlights that execution flow is divided into branches. `if` statements creates branches that can be combined.